bazel/deps.bzl

```diff
@@ -98,10 +98,10 @@ def stratum_deps():
             commit = "ca0291420b5b47fa2596a00877d1713aab61dc7a",
         )
 
-    if "com_github_p4lang_PI_bf" not in native.existing_rules():
-        # ----- PI for Barefoot targets -----
+    if "com_github_p4lang_PI_bf_8_9" not in native.existing_rules():
+        # ----- PI for Barefoot SDE 8.9.x targets -----
         remote_workspace(
-            name = "com_github_p4lang_PI_bf",
+            name = "com_github_p4lang_PI_bf_8_9",
```

Comment:
Is PI not required after 8.9? Why name it as PI_bf_8_9?
Can I run stratum on the older version ex: 8.9.1 as well if required?

Reply:
SDE 9.0.0 and bmv2 uses default PI commit

Reply:
I didn't add 8.9.1 here. Any reason we need to support 8.9.1?

Reply:
Actually both 8.9.1 and 8.9.2 uses same PI library